cfe_internal/enterprise/federation/federation.cf

```diff
@@ -153,12 +153,20 @@ bundle agent transport_user
           "$(ssh_known_hosts)",
           "$(ssh_config)"
         };
+      "ssh_home_cil_file" string => "/tmp/cftransport_ssh_home_t.cil";
```

Comment:
This file will be left in place when no longer needed, right? Maybe we should put it somewhere into one of "our" folders?

Reply:
Yes. It would be leftover in /tmp. We could certainly put it in /var/cfengine/state or /opt/cfengine/federation maybe?